src/BUILD

```diff
@@ -559,18 +558,19 @@ cc_binary(
     linkstatic = True,
 )
 
-cc_import(
-  name = "shared_lib",
-  hdrs = ["pocapi.h"],
-  shared_library = "ovms_shared",
-)
-
-cc_library(
```

Comment:
Shouldn't we leave this for benchmark app?

Reply:
done